README.md

```diff
@@ -169,43 +185,92 @@ target = "binary"
 
 # Creating GraphDataset objects
 dataset_train = GraphDataset(
-    hdf5_path = "<train_hdf5_path.hdf5>",
+    hdf5_path = hdf5_paths,
+    subset = train_ids, 
     node_features = node_features,
     edge_features = edge_features,
     target = target
 )
 dataset_val = GraphDataset(
-    hdf5_path = "<val_hdf5_path.hdf5>",
+    hdf5_path = hdf5_paths,
+    subset = valid_ids, 
     node_features = node_features,
     edge_features = edge_features,
     target = target
 
 )
 dataset_test = GraphDataset(
-    hdf5_path = "<test_hdf5_path.hdf5>",
+    hdf5_path = hdf5_paths,
+    subset = test_ids, 
     node_features = node_features,
     edge_features = edge_features,
     target = target
 )
 ```
 
+#### GridDataset
+
+For later CNNs training. 
+
+```python
+from deeprankcore.dataset import GridDataset
+
+features = ["bsa", "res_depth", "hse", "info_content", "pssm", "distance"]
+target = "binary"
+
+# Creating GraphDataset objects
+dataset_train = GridDataset(
+    hdf5_path = hdf5_paths,
+    subset = train_ids, 
+    features = features,
+    target = target
+)
+dataset_val = GridDataset(
+    hdf5_path = hdf5_paths,
+    subset = valid_ids, 
+    features = features,
+    target = target
+
+)
+dataset_test = GridDataset(
+    hdf5_path = hdf5_paths,
+    subset = test_ids, 
+    features = features,
+    target = target
+)
+```
+
 ### Training
 
-Let's define a Trainer instance, using for example of the already existing GNNs, GINet:
+Let's define a Trainer instance, using for example of the already existing GNNs, GINet. Note that being GINet a GNN, it expects datasets' instances of type `GraphDataset`.
```

Comment:
```suggestion
Let's define a Trainer instance, using for example of the already existing GINet. Because GINet is a GNN, it requires a dataset instance of type `GraphDataset`.
```